backend/main.py

```
@@ -76,6 +73,45 @@ def authenticate_user(db: Session, username: str, password: str):
         return False  # Password is incorrect  
     return user  # User is authenticated  
 
+from fastapi import BackgroundTasks
```

Ruff (0.8.2)

Module level import not at top of file

(E402)